Content.Server.Database/Model.cs

```diff
@@ -500,6 +501,20 @@ public sealed class ServerRoleUnban
         public DateTime UnbanTime { get; set; }
     }
 
+    [Table(nameof(PlayerId))]
+    public sealed class RoleTimer
+    {
+        [Required, Key, DatabaseGenerated(DatabaseGeneratedOption.Identity)]
+        public int Id { get; set; }
+
+        [Required, ForeignKey("player")]
+        public Guid PlayerId { get; set; }
+
+        public string Role { get; set; } = string.Empty;
```

Comment:
```suggestion
        public string Role { get; set; } = null!;
```

An empty string would still be an invalid value. The only difference is that the nullability system isn't as loud about it, but that should be ignored.